### ISSUE TITLE
Coloring of Nested Containers in Sequence Diagrams fails

### ISSUE THREAD

**Issue by RolfBremer** [hikerpig/pintora-vscode#8]
If I have two or more nested containers, lets say a "loop" and an "alt" inside of it, and I want the alt to have a green background, and the outer "loop" has a blue one, the inner container also gets blue:

```
    loop #blue All .pintora files in pics folders
        Provider->>Provider: Check if pintora file\n is newer than korresponding SVG File
        alt #green Found change
            Provider->>Pintora: Compile to SVG
            activate Pintora
            Pintora-->>Provider: Provide SVG File
            deactivate Pintora
            Provider-->>TypstProject: Provide SVG File
        end
    end
```

![image](https://github.com/hikerpig/pintora-vscode/assets/46456966/7cf975e3-3f25-4021-855d-bc5a50b170dd)

**Comment by hikerpig** [comment]
This seems to be a bug , I'll see what I can do this week

**Comment by hikerpig** [comment]
This is fixed in v0.9.2
